.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+go:
+        - 1.3
```

Comment:
Oh, OK - I didn't realise that. In fact, I've been compiling and running vulcand with 1.3! What is the reason for not supporting 1.3?